v5/pivotal/projects.go

```diff
@@ -116,3 +116,23 @@ func (service *ProjectService) Get(projectId int) (*Project, *http.Response, err
 
 	return &project, resp, err
 }
+
+func (service *ProjectService) AddStory(
```

Comment:
Hmm, yeah there's a ton of commits in there. I guess from trying to get up to date with your latest master. I actually added another ListLabels func on Projects as well. I'll try to make a new PR with both of them.